src/Nuget/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.Nuspec

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.VisualStudio.Editors</id>
+    <summary>Editors implementation for Project Properties</summary>
+    <description>
+      This is the editors implementation of the Project properties page in VisualStudio
```

Comment:
Again unclear - Maybe something like "Implementation of some designers and project property page editors in VisualStudio"